examples/js/loaders/EXRLoader.js

```diff
@@ -15,6 +15,485 @@ THREE.EXRLoader.prototype = Object.create( THREE.DataTextureLoader.prototype );
 
 THREE.EXRLoader.prototype._parser = function ( buffer ) {
 
+	const USHORT_RANGE = (1 << 16);
```

Comment:
Why does changes to the `EXRLoader` popping up?

Reply:
Sorry about this, my git history has gotten a bit messed up, please ignore while I fix up the history.